tests/_utils/__init__.py

```diff
@@ -25,7 +25,6 @@
 
 import numpy as np
 import pytest
-
```

Comment:
Is this intentional? I thought our style requires a blank line between third and first party imports?

Reply:
If it passes checks, and other files aren't adversely affected, I'd say it's probably good.